### DISCUSSION TITLE
Fixed arm lengths

### DISCUSSION THREAD

**Discussion by bannus** [doudar/SmartSpin2k#474] [Q&A] [answered]
Does anyone have the info for recommended arm lengths for common bikes? It looks like there's supposed to be a table in the README in the "Arm" directory but it's just got a placeholder:

https://github.com/doudar/SmartSpin2k/tree/develop/Hardware/Common%20Assets/Arm

**Comment by doudar** [comment]
Someone (wink wink) should be making a table. 45mm to 60 mm is the sweet spot for most bikes. Since there’s a pivot point on each end there’s a good amount of variation on sizes that will fit. What bike are you building for?

**Reply by eMadman** [reply]
It largely depends on your bike - If you have an IC4, Echelon Connect Sport, or Peloton, I have resizeable ones available on my fork here:
https://github.com/eMadman/SmartSpin2k/tree/adjustable-arm/Hardware/MODS/Adjustable%20Arm

I generally ship size-adjustable ones to allow for some inaccuracy and variability in measurements and mounting locations.  Since you're printing your own, measuring and sizing a fixed length arm gives you the strongest possible solution. The resizeable ones are pretty tough to break.

**Reply by bannus** [reply]
@doudar I'm building for a Bowflex C6 (IC4).

**Reply by eMadman** [reply]
the 60-70mm range should have you in the sweet spot.  I'd suggest starting with the 70 since it allows more flexibility - you can always move the bike mount up or down the front tube if it's too long.  If it's too short, the unit might be at a slight angle to the knob.  Still functional but may cause some rubbing on the material.

**Comment by doudar** [comment, marked as answer]
> @doudar I'm building for a Bowflex C6 (IC4).

Okay - if you get the same measurement as this photo below (Middle of the knob to the back of the head tube), I'd say a 65mm fixed arm should work. (The assembly below is has a 70mm on it). I'd love feedback on it so we can get that table built. We used to use all adjustable arms but my thought is that an adjustable arm is basically two fixed arms bolted together, so if you know the correct length for a fixed arm, you'll get a better arm (more stable) with less parts. The arm can normally be a little longer than the straight line measured distance as long as your handlebars aren't at minimum height. 

![image](https://github.com/doudar/SmartSpin2k/assets/17362216/70f19f41-636a-4559-8f16-235a4bd04fd3)
![image](https://github.com/doudar/SmartSpin2k/assets/17362216/9097cb12-f144-4785-9707-bd6d269d2c64)
![image](https://github.com/doudar/SmartSpin2k/assets/17362216/3176f302-0a5c-44c8-bd97-fd2b3a902010)

**Reply by bannus** [reply]
Thanks! I'll try 65mm and let you know how it goes.

**Reply by gtagfan13** [reply]
Very 👍👍👍 good

**Comment by gtagfan13** [comment]
Good
